### ISSUE TITLE
TestRunner_quiescence fails occasionally

### ISSUE THREAD

**Issue by mainred** [hashicorp/consul-template#1155]
Please note that the Consul Template issue tracker is reserved
for bug reports and enhancements. For general usage questions,
please use the Consul mailing list:

    https://groups.google.com/forum/#!forum/consul-tool

### Consul Template version

Run `consul-template -v` to show the version. If you are not
running the latest version, please upgrade before submitting an
issue.

### Configuration

```hcl
# Copy-paste your Consul Template configuration file here
```

```liquid
# Copy-paste your Consul Template template here
```

### Command

```shell
# Place your Consul Template command here
```

### Debug output

Provide a link to a GitHub Gist containing the complete debug
output by running with `-log-level=trace`.

### Expected behavior

What should have happened?
Unit tests all pass.

### Actual behavior

What actually happened?
Ci is broken for failed template, 

--- FAIL: TestRunner_quiescence (0.00s)
    --- FAIL: TestRunner_quiescence/snooze (0.07s)
    	runner_test.go:879: bad duration  0.000009
### Steps to reproduce
By executing `make test`, but it does not happen every time.

### References

Failed CI link is here:
https://travis-ci.org/hashicorp/consul-template/builds/294963360#L929

**Comment by eikenb** [comment]
Hey @mainred, thanks for the report.

I've fixed several race conditions in the tests and haven't seen one of these yet. So it might be fixed. If not I'm already planning on refactoring the runner code which would change the underlying problem code anyways.

TLDR; I'm going to close this because it's either fixed or will change.
